api/v3/common_types.go

```diff
@@ -122,10 +127,12 @@ type CommonSplunkSpec struct {
 
 	// ReadinessInitialDelaySeconds defines initialDelaySeconds(See https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) for Readiness probe
 	// Note: If needed, Operator overrides with a higher value
+	// +kubebuilder:validation:Minimum=0
```

Comment:
is it optional by default? OR do we need an explicit market for that?

Reply:
its mandatory field, if its set less than 0 than it should throw 400 error when tried to create CR. CR creation should fail